setup.html.haml

```diff
@@ -260,11 +265,15 @@ author: Dominik Dary
             preSessionAdbCommands
           %dd
             Specify a list of adb commands that will be executed on the target device before <em>selendroid-server</em> will be started. E.g. <code>shell setprop name selendroid</code>, please note that the adb command itself and the serial will be added by selendroid automatically.
+          %dt
+            model
+          %dd
+            Specify the model (e.g. <code>Nexus One</code>, <code>Nexus 7</code>, <code>Galaxy S3</code>) of an emulator when selecting a device.
       %section#androidDevices
         %h2 Devices
         %p Selendroid supports testing on hardware devices as well as using Android emulators. In the capabilities the properties are used to find the device for the test execution. The supported properties in the capabilities are listed <a href="#DesiredCapabilities">here</a>. 
         %h3 Example for selecting a device
-        %p For running a test on the Android target platform 18 and on an emulator the capabilities are for the selendroid-test-app:
+        %p For running a test on the Android target platform 18 and on an emulator which simulating a Nexus 7, the capabilities for the selendroid-test-app are:
```

Comment:
you can either leave out the the word "which" in "which simulating a Nexus 7" or insert the word "is" --> "which is simulating a Nexus 7"